local/components/codecraft/null/templates/.default/component_epilog.php

```diff
@@ -0,0 +1,23 @@
+<? if (!defined('B_PROLOG_INCLUDED') || B_PROLOG_INCLUDED !== true) {
+    die();
+}
+
+/**
+ * @var CCodeCraftNullComponent $this
+ *
+ * @var array                   $arParams
+ * @var array                   $arResult
+ * @var array                   $templateData
+ *
+ * @var string                  $componentPath
+ * @var string                  $epilogFile
+ * @var string                  $templateName
+ * @var string                  $templateFile
+ * @var string                  $templateFolder
+ *
+ * @global CUser                $USER
+ * @global CMain                $APPLICATION
+ * @global CDatabase            $DB
+ */
+
+$APPLICATION->SetTitle($arResult['TITLE']);
```

Comment:
Yep. :-)
arResult array key must contains in SetResultCacheKeys method parameters to save in cache. This key do not included into cached keys, and do not saved in cache.
You can check it.